src/main/java/sg/christopher/duke/entities/Task.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Wrapper class for tasks.
+ */
+package sg.christopher.duke.entities;
+
+import java.io.Serializable;
+
+public class Task implements Serializable {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Creates a new Task.
+     *
+     * @param description description of the task
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Gets a tick or cross representation of the done status of the task.
+     *
+     * @return tick or cross icon representation of the done status of the task
+     */
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
One small suggestion: I think you can remove the comment since you've already mentioned what this function returns in Javadocs.